### ISSUE TITLE
std::cout << tab really slow?

### ISSUE THREAD

**Issue by nickhuangxinyu** [p-ranav/tabulate#74]
Hi, tabulate is very beatiful.

but compare with plain cout, it seems much slower.

my plain cout cost 0.002s but after i use tabulate, it cost 0.05 about 25 times comparing with plain.

Is there any method to speed up?

ps. i always use tabulate to show a stl map.
using table, add key row and column row, set some show style.

Did i miss anything?

thanks

**Comment by seanmamasde** [comment]
yeah it's really slow in my experience, if you're dealing with large amount of rows you can literally see the console output being produced, as standard output finished printing everything in one go.
